Authentication/SignUp/View/SignupController.swift

```diff
@@ -6,14 +6,14 @@
 //  Copyright © 2016 Wolox. All rights reserved.
 //
 
-import Foundation
 import ReactiveCocoa
 
 public final class SignupController: UIViewController {
     
-    private let _viewModel: SignupViewModelType
+    private var _viewModel: SignupViewModelType
```

Comment:
Sorry, that remained from a time where the controller set a property of the view model.